INSTALL.md

```diff
@@ -43,130 +37,123 @@ If you are building ATF from the code on the repository, you will also need
 to have GNU autoconf, automake and libtool installed.
 
 
-Regenerating the build system
-*****************************
+## Regenerating the build system
 
 If you are building ATF from code extracted from the repository, you must
 first regenerate the files used by the build system.  You will also need to
-do this if you modify configure.ac, Makefile.am or any of the other build
+do this if you modify `configure.ac`, `Makefile.am` or any of the other build
 system files.  To do this, simply run:
 
-  $ autoreconf -i -s
+    $ autoreconf -i -s
 
-For formal releases, no extra steps are needed.
+For release builds, no extra steps are needed.
 
 
-General build procedure
-***********************
+## General build procedure
 
 To build and install the source package, you must follow these steps:
 
 1. Configure the sources to adapt to your operating system.  This is done
-   using the 'configure' script located on the sources' top directory,
+   using the `configure` script located on the sources' top directory,
    and it is usually invoked without arguments unless you want to change
    the installation prefix.  More details on this procedure are given on a
    later section.
 
 2. Build the sources to generate the binaries and scripts.  Simply run
-   'make' on the sources' top directory after configuring them.  No
+   `make` on the sources' top directory after configuring them.  No
    problems should arise.
 
-3. Install the program by running 'make install'.  You may need to become
+3. Install the program by running `make install`.  You may need to become
    root to issue this step.
 
 4. Issue any manual installation steps that may be required.  These are
    described later in their own section.
 
-5. Check that the installed programs work by running 'make installcheck'.
+5. Check that the installed programs work by running `make installcheck`.
    You do not need to be root to do this, even though some checks will not
    be run otherwise.
 
 
-Configuration flags
-*******************
+## Configuration flags
 
-The most common, standard flags given to 'configure' are:
+The most common, standard flags given to `configure` are:
 
-* --prefix=directory
-  Possible values: Any path
-  Default: /usr/local
+* `--prefix=directory`::
+  **Possible values:** Any path
+  **Default:** `/usr/local`
```

Comment:
_(picking a line since this is a common proposed pattern in this change)_
This is more difficult to read rendered (all of the text is jammed on to a single line). I think separating them out on different lines is better.